testsuite/tests/ui/test_service_discovery.py

```diff
@@ -0,0 +1,32 @@
+"""Test for service discovery"""
+import pytest
+
+from testsuite import settings, resilient
+from testsuite.ui.views.admin.product.product import ProductNewView
+
+
+# pylint: disable=unused-argument, too-many-arguments
+@pytest.fixture(scope="module")
+def service(login, openshift, navigator, threescale, testconfig, request):
+    """Discover service in OpenShift"""
+    namespace = settings["openshift"]["projects"]["threescale"]["name"]
+    openshift().do_action("adm",
+                          ['policy', 'add-cluster-role-to-user', 'view', f'system:serviceaccount:{namespace}:amp'])
```

Comment:
this definitely needs to be done somewhere else since this 1) requires really broad permissions 2) doesn't have to be performed every time the testsuite is run. single invocation during 3scale deployment is enough and desired.